unstructured/ingest/connector/azure_cognitive_search.py

```diff
@@ -150,8 +150,6 @@ def write(self, docs: t.List[BaseSingleIngestDoc]) -> None:
             local_path = doc._output_filename
             with open(local_path) as json_file:
                 json_content = json.load(json_file)
-                for content in json_content:
-                    self.conform_dict(data=content)
```

Comment:
How will ingest conform the dicts now since we remove it from write methods?

Reply:
This was moved to the base classes in the destination connector, meaning whenever you call `write()` or `write_elements` not, it'll call the `conform_dict` method on your behalf.